client/js/components/statement/listOriginalStatements/ListOriginalStatements.vue

```diff
@@ -0,0 +1,565 @@
+<template>
+  <div>
+    <dp-map-modal
+      ref="mapModal"
+      :procedure-id="procedureId" />
+
+    <dp-loading
+      v-if="isLoading"
+      class="u-mt"/>
+
+    <template v-else>
+      <dp-pager
+        v-if="pagination.currentPage && originalStatements.length > 0"
+        :class="{ 'invisible': isLoading }"
+        class="mt-4 mb-2"
+        :current-page="pagination.currentPage"
+        :total-pages="pagination.totalPages"
+        :total-items="pagination.total"
+        :per-page="pagination.perPage"
+        :limits="pagination.limits"
+        @page-change="fetchOriginalStatementsByPage"
+        @size-change="handleSizeChange"
+        :key="`pager1_${pagination.currentPage}_${pagination.perPage}`" />
+
+      <dp-data-table
+        v-if="originalStatements.length > 0"
+        has-flyout
+        :header-fields="headerFields"
+        is-expandable
+        :items="originalStatements"
+        track-by="id">
+        <template v-slot:externId="{ externId }">
+          <span
+            class="font-semibold"
```

Comment:
Not to be taken into account with this here PR, mostly a note to myself: font weights are sadly not yet usable via Tailwind. at least i am not sure how this will interfere with our font definitions.